exonum-java-binding/CHANGELOG.md

```diff
@@ -19,7 +19,9 @@ The release is based on Exonum 0.11
 
 ### Added
 - `toOptional()` method to `EntryIndexProxy`. (#790)
-- `getPoolTransactionHashes()` method to `Blockchain`. (#850)
+- `getTransactionPool()` method to `Blockchain`. (#850)
+- SLF4J to Log4j binding to enable libraries coded to the SLF4J API to use Log4j 2, 
```

Comment:
I believe that "- SLF4J support" is enough in the changelog. Also, I would mention in docs that EJB provides `log4j 2` as a logging implementation